CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,10 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+== 2025-09-26 Add New Experimental Histogram View
+<p>New Experimental Histogram View added
+<p>Can be found in Experimental->Views->New Histogram
```

Comment:
I wonder whether it would be worth highlighting why the new version of the histogram has been added, as on its own the change might not make much sense